website/react-magma-docs/src/pages/api/icon-button.mdx

```diff
@@ -344,7 +343,7 @@ export function Example() {
 
 Passing in the `testId` prop will add the `data-testid` attribute to the button element for easier querying in tests.
 
-```html
+```html noCodeSandbox
```

Comment:
Should we make sure we consistently use `noCodeSandbox` for ALL Testing sections of the docs?